index.html

```diff
@@ -475,9 +475,12 @@ <h3>The <dfn>PerformanceResourceTiming</dfn> Interface</h3>
 <dfn>workerStart</dfn> attribute MUST return as follows:</p>
 <ol>
 <li>
-<p>If the current browsing or worker context's have an
+<p>If the current browsing or worker context have an
 <a data-cite="service-workers-1#dfn-containing-service-worker-registration">
-active worker</a> [[service-workers-1]]:</p>
+active worker</a> [[service-workers-1]] and if the resource
```

Comment:
nit: perhaps have the check on the destination type first, as that check is cheaper and that is probably how it would be implemented.

Reply:
Note that it's an "or" relationship, so both tests will need to run (unless I misunderstand and you want destination and TAO checks to run before the active SW one).